osutil/user.go

```diff
@@ -60,30 +60,35 @@ func AddExtraSudoUser(name string, sshKeys []string, gecos string) error {
 		return fmt.Errorf("adduser failed with %s: %s", err, output)
 	}
 
-	sudoersFile := filepath.Join(sudoersDotD, "create-user-"+name)
+	sudoersFile := filepath.Join(sudoersDotD, "create-user-"+strings.Replace(name, ".", ",", -1))
```

Comment:
Comma reads incorrectly for me. It feels like two independent things, rather than an escaping sequence.

Let's please use %46 to escape the dot, which should work fine since % is disallowed too.

This also gives us a good pattern to fix further cases, if we find them.